doc/src/explanation/active-deprecations.md

```diff
@@ -0,0 +1,1231 @@
+
+This same pattern may be used to instead turn `SymPyDeprecationWarning` into
+an error so that you can test that you aren't using deprecated code. To do
+this, replace `"ignore"` with `"error"` in the above example. You may
+also omit `message` to make this apply to all `SymPyDeprecationWarning`
+warnings.
+
+If you are using pytest, you can use the [pytest warnings filtering
+capabilities](https://docs.pytest.org/en/latest/how-to/capture-warnings.html)
+to either ignore `SymPyDeprecationWarning` or turn them into errors.
+
+```{note}
+The Python [`-W`
+flag](https://docs.python.org/3/using/cmdline.html#cmdoption-W) and
+[`PYTHONWARNINGS` environment
+variable](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONWARNINGS)
+will NOT work to filter SymPy deprecation warnings (see [this blog
+post](https://nedbatchelder.com/blog/201810/why_warnings_is_mysterious.html)
+by Ned Batchelder and [this SymPy
+issue](https://github.com/sympy/sympy/issues/15130) for details on why). You
+will need to either add a `warnings` filter as above or use pytest to filter
+SymPy deprecation warnings.
+```
+
+## Version 1.10
+
+(deprecated-traversal-functions-moved)=
+### Some traversal functions have been moved
+
+Some traversal functions have moved. Specifically, the functions
+
+- `bottom_up`
+- `interactive_traversal`
+- `postorder_traversal`
+- `preorder_traversal`
+- `use`
+
+have moved to different SymPy submodules.
+
+These functions should be used from the top-level `sympy` namespace, like
+
+```py
+sympy.preorder_traversal
+```
+
+or
+
+```py
+from sympy import preorder_traversal
+```
+
+In general, end-users should use the top-level `sympy` namespace for any
+functions present there. If a name is in the top-level namespace, its specific
+SymPy submodule should not be relied on, as functions may move around due to
+internal refactorings.
+
+(sympy-core-trace-deprecated)=
+### `sympy.core.trace`
+
+The trace object `sympy.core.trace.Tr()` was moved to
+`sympy.physics.quantum.trace.Tr()`.  This was because it was only used in the
+`sympy.physics.quantum` submodule, so it was better to have it there than in
+the core.
+
+(deprecated-sympy-core-compatibility)=
+### The `sympy.core.compatibility` submodule
+
+The `sympy.core.compatibility` submodule is deprecated.
+
+This submodule was only ever intended for internal use. Now that SymPy no
+longer supports Python 2, this module is no longer necessary, and the
+remaining helper functions have been moved to more convenient places in the
+SymPy codebase.
+
+Some of the functions that were in this module are available from the
+top-level SymPy namespace, i.e.,
+
+```py
+from sympy import ordered, default_sort_key
+```
+
+The remaining were only intended for internal SymPy use and should not be used
+by user code.
+
+Additionally, these two functions, `ordered` `default_sort_key`, used to also
+be in `sympy.utilities.iterables` but have been moved from there as well.
+
+
+## Version 1.9
```

Comment:
Question for @oscarbenjamin before I do it. Are you opposed to adding the date for each release here, like

```
## Version 1.9 (2021-09-30)
```

This would be useful considering we are going to use a date-based policy, but it would also be one more thing to do for the release.